rest-server/src/main/java/org/opennms/horizon/server/service/GrpcNodeService.java

```diff
@@ -216,4 +247,47 @@ private static TopNResponse generateDownloadableTopNResponse(
         }
         throw new IllegalArgumentException("Invalid download format" + downloadFormat.value);
     }
+
+    private static SnmpInterfaceResponse generateDownloadableSnmpResponse(
+            List<SnmpInterfaceDTO> snmpInterfaceDTOS, DownloadFormat downloadFormat) throws IOException {
+        if (downloadFormat == null) {
+            downloadFormat = DownloadFormat.CSV;
+        }
+        if (downloadFormat.equals(DownloadFormat.CSV)) {
+            StringBuilder csvData = new StringBuilder();
+            var csvformat = CSVFormat.Builder.create()
+                    .setHeader(
+                            "ID",
+                            "Node Id",
+                            "Alias",
+                            "Description",
+                            "Admin status",
+                            "Index",
+                            "Name",
+                            "Tenant id",
+                            "Operator status",
+                            "Speed",
+                            "Type")
```

Comment:
```suggestion
                    .setHeader(
                            "ALIAS",
                            "IP ADDR",
                            "PHYSICAL ADDR",
                            "INDEX",
                            "DESC",
                            "TYPE", 
                            "NAME",
                            "SPEED",
                            "ADMIN STATUS",
                            "OPERATOR STATUS")
```
Align it with UI, sent you screenshot with real dadta

Reply:
resolved.added like UI screen shot.